### PR TITLE
MCS-504 - Fixing analysis UI page for Eval 3

### PR DESCRIPTION
**Pull request by rartiss55**

### CODE REVIEW

File: mcs_docker_setup/analysis-ui/src/components/App/header.jsx [hunk 1]
```diff
@@ -51,18 +56,52 @@ class ListItem extends React.Component {
...
-                } else {
-                    params = "?test_type=" + this.props.item;
+            let hasEvalState = this.props.state["eval"] !== undefined && this.props.state["eval"] !== null;
+            let hasSceneNumState = this.props.state["scene_num"] !== undefined && this.props.state["scene_num"] !== null;
+            let hasTestTypeState = this.props.state["test_type"] !== undefined && this.props.state["test_type"] !== null;
+            let hasCatTypeState = this.props.state["category_type"] !== undefined && this.props.state["category_type"] !== null;
+            let paramsToAppend = '';
+
+            if(this.props.stateName === 'eval') {
+                // TODO: MCS-504
```
Comment (brianpippin) [inline mcs_docker_setup/analysis-ui/src/components/App/header.jsx:66]: Is this a todo for this ticket or a future one?

Comment (rartiss55) [reply mcs_docker_setup/analysis-ui/src/components/App/header.jsx:66]: I think this was made while I was trying to get this working...I'll update it for clarity in a sec.

File: mcs_docker_setup/node-graphql/server.schema.js [hunk 2]
```diff
@@ -195,7 +223,12 @@ const mcsResolvers = {
                 .toArray().then(result => {return result});
         },
         getFieldAggregation: async(obj, args, context, infow) => {
-            return await mcsDB.db.collection('msc_eval').distinct(args["fieldName"]).then(result => {return result});
+            if(args["eval"]) {
+                return await mcsDB.db.collection('msc_eval').distinct(args["fieldName"], {"eval": args["eval"]}).then(result => {return result});
+            } else {
+                return await mcsDB.db.collection('msc_eval').distinct(args["fieldName"]).then(result => {return result});
+            }
+            //return await mcsDB.db.collection('msc_eval').distinct(args["fieldName"], {"eval": args["eval"]}).then(result => {return result});
```
Comment (brianpippin) [inline mcs_docker_setup/node-graphql/server.schema.js:231]: You can probably remove all these lines you just have commented out.

Comment (rartiss55) [reply mcs_docker_setup/node-graphql/server.schema.js]: Yup just did :)

**Review by brianpippin — APPROVED** [review]

**Pull request merged**